client/app/components/BottomTabs.js

```diff
@@ -68,20 +68,16 @@ class BottomTabs extends Component {
 }
 
 const tabStyles = StyleSheet.create({
-    container2: {
+    container: {
         position: 'absolute',
         bottom: 0,
     },
-    container: {
-        flex: 1,
-        justifyContent: 'flex-end',
-    },
     bottomTabs: {
         flexDirection: 'row',
         justifyContent: 'space-evenly',
     },
     touchables: {
-        width: '33.3%',
+        width: '33.33333%',
```

Comment:
Mmmm precision

Reply:
This was actually important hahaha cuz there was a sliver of white space at the end of the bottom tabs bc it wasn't adding up to 100% lol so this got rid of that